pkg/framework/http/router/router.go

```diff
@@ -0,0 +1,39 @@
+package router
+
+import (
+	"net/http"
+)
+
+type RouterInterface interface {
+	RegisterRoutes(routes *[]Route)
+	ServeHTTP(w http.ResponseWriter, req *http.Request)
+}
+
+type Router struct {
+	routes map[string]http.HandlerFunc
+}
+
+type Route struct {
+	Path    string
```

Comment:
can you please describe how we define HTTP methods??

Reply:
internal_module/internal_module.go